recipes/exchangelib/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "exchangelib" %}
+{% set version = "3.1.1" %}
+{% set sha256 = "33e76159fa14e004ae8333ae0a0d2bd9c6d4f682f9f63ad659c058ecadb20fef" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: True  # [py<35]
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - cached-property
+    - defusedxml >=0.6.0
+    - dnspython >=1.14.0
+    - isodate
+    - lxml >3.0
+    - oauthlib
+    - pygments
+    - python
+    - python-dateutil
+    - pytz
+    - requests >=2.7
+    - requests_ntlm >=0.2.0
+    - requests-oauthlib
+    - tzlocal
+
+test:
+  imports:
+    - exchangelib
+
+about:
+  home: https://pypi.org/project/exchangelib/
+  license: BSD-2-Clause
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Client for Microsoft Exchange Web Services (EWS)'
+  doc_url: https://pypi.org/project/exchangelib/
```

Comment:
I would use `https://github.com/ecederstrand/exchangelib` for `doc_url` as well.

Reply:
Done